docs/source/model_design/risk_factors.rst

```diff
@@ -200,6 +200,109 @@ Confounding
 Intermediates
 """""""""""""
 
+An intermediate variable as discussed in this section is defined as a variable 
+between an exposure and outcome in a sequential causal chain, as demonstrated 
+in the diagram below:
+
+.. image:: basic_int_diagram.svg
+
+An Example of an Uncomplicated Intermediate
+*******************************************
+
+A (simplified) example that can demonstrate such a causal pathway is the 
+relationship between the exposure of cigarette smoking, which is causally 
+related to the intermediate variable of accumulated tar in lungs, which in 
+turn is causally related to lung cancer (note: for the purposes of this 
+example, assume that there is no direct causal relationship between cigarette 
+smoking and lung cancer other than through the intermediate variable of 
+accumulated tar in lungs).
+
+.. image:: smoking_intermediate_example.svg
+
+Now, let's say that we plan to enact an advertising campaign to reduce smoking 
+in hopes of reducing population lung cancer rates. Our expected causal pathway 
+would look like this:
+
+.. image:: smoking_intervention_example.svg
+
+Therefore, as we increase intervention coverage, we would exect a decrease in 
+population lung cancer rates, as these two variables are located on the same 
+causal pathway. 
+
+An Example of a Complicated Intermediate
+****************************************
+
+Now, let's imagine that a brand new hypothetical medication was just 
+created that dissolves tar in lungs that accumulates due to smoking and is now 
+widely used in some areas. Our causal diagram would now look like this:
+
+.. image:: medication_example.svg
+
+Quickly, we can now see that the relationship we previously knew between 
+smoking and lung cancer is now impacted by the use of this hypothetical 
+medication that affects the intermediate variable between smoking and lung 
+cancer. Therefore, now we can only expect our intervention to decrease 
+population lung cancer rates only in areas that do not widely use this 
+medication.
+
+Notably, intermediate variables may be relevant in situations in which the 
+variable located most proximal to the outcome in the causal pathway is 
+difficult to measure. For instance, measuring the amount of tar in a person's 
+lungs is an invasive procedure; measuring the amount of cigarettes a person 
+smokes is much easier in comparison. Therefore, data availability may dictate 
+that we model cigarette smoking rather than lung tar. 
+
+  This may be a reasonable approach, given that the intermediate variable 
+  *does not* interfere with the relationship between the exposure and outcome 
+  for the given model population (little or no use of the medication that 
+  removes tar from lung). However, if there *is* interference by the 
+  intermediate variable on the relationship between the exposure and outcome 
+  (significant use of the medication), then the previously measured 
```

Comment:
I think this is important to keep because there still will be *some* association between cancer and smoking (even if it is zero), but it will not be the same as what it was when it was measured in the absence of the medication.